src/implementations/twig/components/gallery/gallery-item.html.twig

```diff
@@ -19,6 +22,7 @@
 
 {# variables #}
 
+{% set _id = id|default('') %}
```

Comment:
but we are not generating any id as we say in the doc comment. I would use random() also here, then we will change them all at once.